openy.info.yml

```diff
@@ -65,7 +65,6 @@ dependencies:
   - simple_menu_icons
   # Open Y
   - advanced_help_block
-  - logger_entity
```

Comment:
keep it, please

Reply:
It not used in Open Y code anymore, so no need to enable it on fresh Open Y installation.